README.md

```diff
@@ -9,3 +9,29 @@ This is a project by us, for us, based on the things we like. If you like some o
 ## License
 
 This project is licensed under a 2-clause BSD license.
+
+## Building
+
+The easiest way to build is to use `flatpak-builder` to create a flatpack dev environment.
+
+On a Debian-based system like Ubuntu, the steps will look like below:
+
+```sh
+# Install flatpak if it's not already
+sudo apt install flatpak flatpak-builder
+
+# Re-log your user session after installing flatpak. You should see something similar to the following in ``env | grep XDG``
+#   XDG_DATA_DIRS=$HOME/.local/share/flatpak/exports/share:/var/lib/flatpak/exports/share:<os-defaults>
+
+# Make sure that the gnome-nightly repositories are in your user install directory
+flatpak remote-add --user --if-not-exists gnome-nightly https://nightly.gnome.org/gnome-nightly.flatpakrepo
+
+# Install the GNOME SDK
+flatpak install --user org.gnome.Platform//master org.gnome.Sdk//master
+
+# Build and install the application to your local flatpak install
+flatpak-builder --user --install --force-clean --ccache build org.serenityos.Ladybird-gtk4.json 
+
+# Run the app
+flatpack run --user org.serenityos.Ladybird-gtk4
+```
```

Comment:
I'd like to rewrite these instructions a bit, for one thing to fix the "flatpack" typos, but also to make it clearer which part is about installing Flatpak onto your system, and which part is about building Ladybird.

How do we handle this? Are you OK with me just pushing to your branch?

Reply:
Sure, you can push to the branch, that's why the checkmark is set to allow edits by maintainers

Reply:
I'll give it a whirl here first though, maybe I'll be able make it look like you were thinking?

Reply:
pushed up some README.md refactoring, but feel free to tweak it further before merging (or re-writing bits in a follow up :shrug: )